Script/item.dat

```diff
@@ -823,6 +931,9 @@ holybanana /* materialcontainer->banana-> */
   CanBeMirrored = true; /* will create a peel */
   AllowEquip = true;
   IsValuable = true;
+  CanBeWished = false;
```

Comment:
Should be allowed to wish for one

Reply:
I have forbidden wishing for all artifacts on the reasoning that they should be unique/rare items that you should be excited about finding and they should make you consider your choices and adapt to its use, rather than wishing for any one you like. This stands for artifact weapons and armor, IMHO, but should I make consumable artifacts wishable again? I'm not opposed to that change, but for consistency, maybe it should be all consumable artifacts (the banana, the saga, the monograph)?